lib/creator/transformations/index.js

```diff
@@ -11,9 +17,30 @@
 * @returns { <Void> } TODO
 */
 
-module.exports = function runTransform(transformObject) {
+const transformsObject = {
+	entryTransform,
+	outputTransform
+};
```

Comment:
Don't review yet, will notify when there's something better to review, this isn't nearly done at all